pennylane_lightning/core/src/simulators/lightning_gpu/MPIWorker.hpp

```diff
@@ -175,8 +175,11 @@ make_shared_mpi_worker(custatevecHandle_t handle, MPIManager &mpi_manager,
                                             communicatorType, nullptr);
     if (err != CUSTATEVEC_STATUS_SUCCESS) {
         communicator = nullptr;
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCommunicatorCreate(
-            handle, &communicator, communicatorType, "libmpi.so"));
+        auto py_err = custatevecCommunicatorCreate(
+            handle, &communicator, communicatorType, "libmpi.so");
+        PL_ABORT_IF_NOT(py_err == CUSTATEVEC_STATUS_SUCCESS,
+                        "MPI communicator creation failed. Please add "
```

Comment:
This looks good, but is there a way this could be tested from Python?

Reply:
There is no trivial way to get it tested since it requires a incorrect env setup to get it test.